firefox-ios/firefox-ios-tests/Tests/PerformanceTestPlan.xctestplan

```diff
@@ -98,6 +98,9 @@
         "SyncUITests",
         "TabCounterTests",
         "TabTrayTests",
+        "TabsTests",
+        "TabsTestsIpad",
+        "TabsTestsIphone",
```

Comment:
TopTabsTests wasn't excluded from PerformanceTestPlan. It should not be run here.